computervision/src/main/java/ComputerVision.kt

```diff
@@ -25,11 +28,11 @@ sealed class Vision : SensorReading {
     class Scenery(val scene: String) : Vision()
     class Facial (val prettyFace: String) : Vision()
     class Emotion (val emotion: String) : Vision()
-    class ObjectDetection (val objects: List<Any>): Vision()
+    class ObjectDetection (val objects: JSONArray): Vision()
 }
 
 sealed class CVInput : SensorInput {
-    class ModeChange(val mode: Int) : CVInput()
+    class ModeChange(val mode: JSONObject) : CVInput()
```

Comment:
Same goes here, plus, we won't send mode changes in JSON, we will use this simple format: `inputType_value`.

Reply:
Excuse me, what is exactly the "same" that goes here?

Reply:
Don't expose `JSONObject` in your public API. or any other tokenization details. your `Reading`s and `Input`s were perfect as they were before this commit.

Reply:
Oh that's a misplaced comment! My bad :p